SingularityUI/app/views/view.coffee

```diff
@@ -49,4 +49,10 @@ class View extends Backbone.View
           next: '<span class="glyphicon glyphicon-chevron-right"></span>'
       $('table.paginated').css('display', 'table');
 
+      searchInput = $('.big-search-box')
+      strLength = searchInput.val().length * 2
```

Comment:
We want to make sure it always ends up at the end, there's some cases where length would fail us (Opera sometimes sees a carriage return as 2 characters)